pkg/apis/certmanager/v1alpha1/types.go

```diff
@@ -179,10 +179,10 @@ type ACMEIssuerDNS01ProviderCloudflare struct {
 // ACMEIssuerDNS01ProviderRoute53 is a structure containing the Route 53
 // configuration for AWS
 type ACMEIssuerDNS01ProviderRoute53 struct {
-	AccessKeyID     string            `json:"accessKeyID"`
-	SecretAccessKey SecretKeySelector `json:"secretAccessKeySecretRef"`
-	HostedZoneID    string            `json:"hostedZoneID"`
-	Region          string            `json:"region"`
+	AccessKeyIDRef     SecretKeySelector `json:"accessKeyIDSecretRef"`
```

Comment:
Can we update this PR to not be a breaking change? i.e. if a user specifies `accessKeyID`, it is still used. But if a user specifies `accessKeyIDSecretRef`, it takes precedence?

Reply:
In a later release we can then add validation to prevent both being set, before removing the old field altogether for 1.0.

Reply:
Sure, I can revert it back to what I [originally had](https://github.com/jetstack/cert-manager/pull/197/files/e90e28c67ddbc8af90bd9df6a0be5828fb612c47#diff-4388b350baa92adeeb5cb67f735cb969).

I think at this point, this is blocked on making sure this is a step in the right direction via discussion in #687 though